source/configure/calls-deployment.rst

```diff
@@ -338,10 +342,35 @@ If you wish to host many calls or calls with a large number of participants, tak
   # Allow to allocate more memory as needed for more control messages that need to be sent for each socket connected
   net.core.optmem_max = 16777216
 
+The rtcd service
+----------------
+
+.. include:: ./calls-rtcd-ent-only.rst
+  :start-after: :nosearch:
+
+The Calls plugin has a built-in `Selective Forwarding Unit (SFU) <https://bloggeek.me/webrtcglossary/sfu/>`__ to route audio and screensharing data. But this SFU functionality can be split from the Calls plugin and hosted on its own, e.g., on its own node in a `kubernetes deployment <#kubernetes-deployments>`__, or its own dedicated server `alongside Mattermost instances <#modes-of-operation>`__ in a cloud service or self-hosted.
+
+Reasons to use the ``rtcd`` service
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It helps to understand when and why your organization would want to use ``rtcd``.
+
+First of all, ``rtcd`` is a standalone service, which adds operational complexity. For most small instances of Mattermost there is no need to use ``rtcd``. Instead, use the Calls plugin until performance or reliability becomes an issue.
```

Comment:
I went back and forth on this, and to keep the language clear (and to avoid repeating ourselves) I thought it would be best to have a "warning/background" type of paragraph first, and then go into the reasons why (instead of mixing the two). @justinegeffen can you go over this (line 358-360) and let me know if you can think of a better way to phrase it?